### PR TITLE
Create a Single-Cell topic

### PR DESCRIPTION
**Pull request by nomadscientist**
The MUSIC tutorial 2.0 begins!

### CODE REVIEW

**Comment by nomadscientist** [comment]
#TODO

- [ ] TRY A WORKFLOW WITH 3 DIFFERENT VARIABLES (does the stats handle this?)
- [ ] Fix main tutorial dataset descriptions
- [ ] Remake workflow with separated scRNA-seq references
- [ ] Future: include link to 'how we made the datasets'

**Comment by nomadscientist** [comment]
OK this has expanded into a major overhaul of the transcriptomics page
Bad move on my part to include it here
But here we are...

**Comment by nomadscientist** [comment]
**Need to:**
- [ ]   Fix main tutorial dataset descriptions
- [ ] beautifully annotated workflows and histories for every step
- [ ] scRNA-seq reference tutorial
- [ ] bulk RNA-seq eset tutorial
- [ ] compare tutorial

**Bonus**
- [ ] QC subsampling tester workflow & tutorial
- [ ] Add blog post about the single cell page
- [ ] Add blog post about the deconvolution tutorials
- [ ]  TRY A WORKFLOW WITH 3 DIFFERENT VARIABLES (does the stats handle this?)

File: topics/single-cell/tutorials/bulk-music-2-preparescref/workflows/part2_workflow.ga [hunk 1]
```diff
@@ -0,0 +1,1060 @@
...
+                                "output_name": "output",
+                                "uuid": "3cefa576-db2d-44f6-8cf2-26c846cf5d88"
+                            }
+                        ]
+                    }
+                },
+                "tags": "",
+                "uuid": "1eccbdc6-6b28-4cbf-abbe-59145eeee57e"
+            },
+            "tool_id": 171553,
```
Comment (shiltemann) [inline topics/single-cell/tutorials/bulk-music-2-preparescref/workflows/part2_workflow.ga:846]: The build error now is related to this, doesn't recognize it as a valid tool, something for us to fix, not you @nomadscientist

Comment (hexylena) [reply topics/single-cell/tutorials/bulk-music-2-preparescref/workflows/part2_workflow.ga:846]: I've pushed a commit that *should* fix this.

Comment (hexylena) [reply topics/single-cell/tutorials/bulk-music-2-preparescref/workflows/part2_workflow.ga:846]: Success! Now failing for a much more reasonable reason:

```
in `render': Could not find document 'topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md' in tag 'link'. (ArgumentError)
```

Comment (shiltemann) [reply topics/single-cell/tutorials/bulk-music-2-preparescref/workflows/part2_workflow.ga:846]: amazing! thanks!

File: topics/single-cell/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+requirements:
+  -
+    type: "internal"
+    topic_name: introduction
+
+maintainers:
+  - bebatut
+  - mblue9
+  - heylf
+  - nomadscientist
```
Comment (shiltemann) [inline topics/single-cell/metadata.yaml:17]: @nomadscientist should maintainers for single-cell topic just be you and @mtekman maybe?

**Review by shiltemann — DISMISSED** [review]

**Review by shiltemann — APPROVED** [review]

**Comment by mtekman** [comment]
amazing work all :rocket:

**Pull request merged**